.github/workflows/ci.yml

```diff
@@ -1,19 +1,15 @@
 name: CI
 on:
   pull_request: {}
-  push:
-    branches:
-    - master
 
 jobs:
   rust_build:
     name: Rust build
     timeout-minutes: 5
     runs-on: ubuntu-latest
     container:
-      image: docker://rust:1.56.0-buster
+      image: docker://rust:1.59.0-buster
```

Comment:
nit, TIOLI: now that Rust 1.60 is out, we may as well bump all the way to the latest Rust?

Reply:
It takes about a day for the images to show up on https://hub.docker.com/_/rust. Not there yet.